Algorithm.Python/CustomDataIndicatorExtensionsAlgorithm.py

```diff
@@ -24,6 +24,7 @@
 from QuantConnect.Data import *
 from QuantConnect.Data.Market import *
 from QuantConnect.Data.Custom import *
+from QuantConnect.DataSource import *
 from QuantConnect.Algorithm import *
 from QuantConnect.Python import PythonQuandl
```

Comment:
Think should replace these from the single import statement `from AlgorithmImports import *`